SmartDeviceLink/SDLChoiceCell.h

```diff
@@ -0,0 +1,82 @@
+//
+//  SDLChoiceCell.h
+//  SmartDeviceLink
+//
+//  Created by Joel Fischer on 5/21/18.
+//  Copyright © 2018 Livio. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class SDLArtwork;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLChoiceCell: NSObject
+
+/**
+ Maps to Choice.menuName. The primary text of the cell.
+ */
+@property (copy, nonatomic, readonly) NSString *text;
+
+/**
+ Maps to Choice.secondaryText. Optional secondary text of the cell, if available.
+ */
+@property (copy, nonatomic, readonly, nullable) NSString *secondaryText;
```

Comment:
Add to documentation that duplicate `secondaryText`s are ok.

Reply:
Done